### DISCUSSION TITLE
Feature request !

### DISCUSSION THREAD

**Discussion by thienit99** [Dhruv-Techapps/acf-docs#351] [Ideas]
I really like your extension. I need these 2 features, if you can, research & add them to help me.
1/ Random from the string of words and choose 1 or 2 or 3 random words from the string of words.
Ex:
<random[Oliver Abigail Alexander Amelia Ava Barbara] {1}>  Amelia 
<random[Oliver Abigail Alexander Amelia Ava Barbara] {2}>  AbigailAva 
<random[Oliver Abigail Alexander Amelia Ava Barbara] {3}>  BarbaraAmeliaAbigail

2/ Save the results to txt if found data when loading the page.
Ex:
Loading page X +++> extract [Value element Y] to file.txt

Thanks so much.

**Comment by KannaSakura** [comment]
I also need these features, and I also need a feature that can be started at a specified time, such as when the time reaches 00:00:00

**Comment by dharmesh-hemaram** [comment]
@CrxHuy 

We have added this feature in DEV version please try out and let us know your thoughts

Value `<random(Hello|Welcome)>` - `Hello` | `World`
Value `<random(Oliver|Abigail|Alexander|Amelia|Ava|Barbara)>` - `Alexander`

Similarly try other combinations

**Comment by dharmesh-hemaram** [comment]
@KannaSakura ,

Go to configuration page and press `F12` on right click and select inspect.

- under console tab paste `localStorage.setItem("DEV",true)` string and press enter
- This will unlock start time under config settings.
![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/2290202/08d150ca-435a-4aca-859d-213e56b533a4)

**Comment by KannaSakura** [comment]
Thank you very much, this helped me a lot

________________________________
